### ISSUE TITLE
[Feature Request] Show single page

### ISSUE THREAD

**Issue by mbnoimi** [sufone/qawl#17]
I suggest adding an option for showing a single page per layout (it could be a switch/radio button)

**Comment by mr-islam** [comment]
Good idea, will be considered after matching the remaining features from the old Qawl (dark theme, quick-return to last page, icons, tooltips)
